template/src/components/layers/IsolineLayer.js

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'react-redux';
+import { GeoJsonLayer } from '@deck.gl/layers';
+
+export default function IsolineLayer() {
```

Comment:
Rename to IsochroneLayer.

Reply:
I've named as Isoline because it can be used to show isodistance, not only isochrones. What do you think?